sbin/create_genesis.sh

```diff
@@ -5,11 +5,12 @@ SBIN="$(
   cd "$SBIN"
   pwd
 )"
-. $SBIN/utils/utils.sh
+. "$SBIN/utils/utils.sh"
 ROOT_DIR=$SBIN/..
 
 # Check that the all required dotenv files exists.
 reqdotenv "paths" ".paths.env"
+source .paths.env
```

Comment:
What's going on here? Doesn't `reqdotenv` already source?

Reply:
This is a debug line I am testing something out. It will be removed